jobs/pullAocLeaderboards2024.py

```diff
@@ -0,0 +1,62 @@
+#!/bin/env python3
+from dotenv import load_dotenv
+import json
+import os
+import requests
+import time
+from pathlib import Path
+
+# Load environment variables from .env.
+load_dotenv()  
+
+YEAR = str(os.getenv('AOC_YEAR'))
+COMPETITION_FILE = os.getenv('COMPETITION_FILE')
+SESSION_TOKEN = os.getenv('AOC_TOKEN')
+
+OUTPUT_FILE_NAME = f"schoolScores{YEAR}.json"
```

Comment:
Can we change the output file name to be `generatedAocLeaderboard2024.json`? Just want to keep it consistent with previous years' naming convention.